package.json

```
@@ -4,6 +4,9 @@
   "description": "Common Markdown table-related utilities",
   "main": "./index.js",
   "types": "./index.d.ts",
+  "engines": {
+    "node": ">=16.13.0"
+  },
```

```suggestion
  "packageManager": "npm",
  "engines": { "node": "^16.13.0", "npm": "^8.1.0", "npx": "^8.1.0" },
  "main": "./index.js",
  "types": "./index.d.ts",
```

Would anyone think it would be a bad idea if we did this instead? I am unsure whether we should reasonably support any Node.js releases that are currently not LTS.